<ui version="4.0" >
 <class>HelpBrowser</class>
 <widget class="QMainWindow" name="HelpBrowser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Sqliteman Help</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item row="0" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="textBrowser" >
       <property name="openExternalLinks" >
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="action_Close" />
   <addaction name="separator" />
   <addaction name="actionBack" />
   <addaction name="actionForward" />
  </widget>
  <action name="actionBack" >
   <property name="text" >
    <string>&amp;Back</string>
   </property>
  </action>
  <action name="actionForward" >
   <property name="text" >
    <string>&amp;Forward</string>
   </property>
  </action>
  <action name="action_Close" >
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
